index.bs

```diff
@@ -788,6 +801,63 @@ partial interface ModelBuilder {
     </div>
 </div>
 
+### instanceNormalization ### {#api-modelbuilder-instancenorm}
+Normalize the input features per feature instance using [[Instance-Normalization]]. Unlike [[#api-modelbuilder-batchnorm]] where the mean and variance values are computed across the batch dimension during training, the mean and variance values of instance normalization is computed from each individual feature instance on the fly.
+<script type=idl>
+dictionary InstanceNormalizationOptions {
+  Operand scale;
```

Comment:
I wonder if it would be better to not allow "Operands" in "Options", and make "Operands" explicit (even if optional). I also think it is useful to restrict "Options" to values that are known at the graph-building time (while "Operands" represent values known at run-time).

Reply:
@gramalingam That's a good observation and I have thought about this exact point but ultimately went for the idea that any optional params, whether it's an operand or not, ought to be placed in the options struct because the original motivation for this is to simplify future changes to the API so that no overload will ever be needed in the future. And, that a future change may end up being an additional operand. 

So even if we are willing to draw an arbitrary line right now barring a non-compile time param from entering the options struct, we will be forced to break that rule anyway in the future at the very first instance of the need to extend the API to support an additional optional operand.

Reply:
"Any optional param ought to be placed in the options struct" seems a bit extreme. In the end, either is functionally equivalent, and it is mostly a matter of style as to which is more readable. I personally prefer making the operand explicit, but am okay with this